### DISCUSSION TITLE
Import warnings

### DISCUSSION THREAD

**Discussion by mintcd** [mdbootstrap/TW-Elements#1708] [Support from community] [answered]
Hi guys, I constantly got warnings like this
"Attempted import error: 'initTE' is not exported from 'tw-elements' (imported as 'initTE')"
I could render TE components, but these warnings really annoyed me. How do I solve this?

**Comment by juujisai** [comment]
Hi, what stack are you using in your app? The `initTE` is not beeing exported in the UMD format of the package, maybe your app is using UMD instead of ES, so it doesn't need `initTE` method to work?

**Comment by LucasDiias** [comment]
Same thing! I started my Next 13 project and soon added tw-elements, I followed the steps described in the documentation and even so I received these import warnings, even repeating exactly what was there. In my case, I followed the whole integration example with Next step by step and it warned all components (InitTE, Datepicker and Input) the same way.

I tried using tw-elements-react but nothing was different

**Comment by juujisai** [comment]
Yes, I've seen those import errors. The components were still working for me. Is this the same case for you? We will try to look into that and see why it's happening.

**Comment by Webtransformer** [comment]
Have the same issues with Next 13.4.9

**Comment by juujisai** [comment, marked as answer]
I do not see those issues anymore when creating a new app with of the latest nuxt template:

I've loaded older next project and the error messages about importing from `tw-elements` and `document not defined` were still there. What worked for me was changing the imports to dynamic imports inside the `useEffect` method. 
```
  useEffect(() => {
    const init = async () => {
      const { Datepicker, Input, initTE } = await import("tw-elements");
      initTE({ Datepicker, Input });
    };
    init();
  }, []);
```

Please give it a try and let me know whether it helped or not.

Best Regards!
